### PR TITLE
Vinay Made some changes

### PR DESCRIPTION
**Pull request by geekvinay**
I have added my resume

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -77,12 +77,16 @@ <h2>Shyam Sundar</h2>
             <img src="./assets/social-icons/github.svg" width="32" height="32" />
           </a>
 
-          <a href="https://www.linkedin.com/in/shyam-sundar-renudevan-228325200/" target="_blank" class="link">
+          <a
```
Comment (PuruVJ) [inline index.html:80]: dude can you run prettier in this file?

**Comment by PuruVJ** [comment]
Dude can you run prettier in the index.html and vinay.html and then push again?

File: resume/vinay.html [hunk 2]
```diff
@@ -0,0 +1,202 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>NAME / Résumé / DBMS Phase 1</title>
```
Comment (PuruVJ) [inline resume/vinay.html]: Can you put your name here?

**Review by Madhav2133 — APPROVED** [review]
merge this asap so i can add abhiram and shyam's resume too

**Review by PuruVJ — APPROVED** [review]

**Pull request merged**